hub/config/src/integration-test/java/uk/gov/ida/integrationtest/hub/config/apprule/IdentityProviderResourceIntegrationTest.java

```diff
@@ -206,27 +219,44 @@ public void getIdpConfigData_returnsNotFoundIfIdpDoesNotExist() {
 
 
     @Test
-    public void getEnabledIdentityProviderEntityIdsForLoa_returnsOkAndIdps() {
+    public void getEnabledIdentityProviderEntityIdsForIdpAuthnRequestGenerationAndLoa_returnsOkAndIdps() {
         String entityId = "not-test-rp";
-        Response response = getIdpListForLoA(entityId, LevelOfAssurance.LEVEL_1, Urls.ConfigUrls.ENABLED_ID_PROVIDERS_FOR_LOA_RESOURCE);
+        Response response = getIdpListForLoA(entityId, LevelOfAssurance.LEVEL_1, Urls.ConfigUrls.ENABLED_ID_PROVIDERS_FOR_REGISTRATION_AUTHN_REQUEST_RESOURCE);
         assertThat(response.getStatus()).isEqualTo(Response.Status.OK.getStatusCode());
 
-        List<String> providerEntityIds = response.readEntity(new GenericType<List<String>>(){});
+        List<String> providerEntityIds = response.readEntity(new GenericType<List<String>>() {
+        });
         assertThat(providerEntityIds).containsOnly(
                 ENABLED_ALL_RP_IDP
         );
     }
 
+    @Test
+    public void getEnabledIdentityProviderEntityIdsForIdpResponseProcessingAndLoa_returnsOkAndIdps() {
+        String entityId = "not-test-rp";
+        Response response = getIdpListForLoA(entityId, LevelOfAssurance.LEVEL_1, Urls.ConfigUrls.ENABLED_ID_PROVIDERS_FOR_REGISTRATION_AUTHN_RESPONSE_RESOURCE);
+        assertThat(response.getStatus()).isEqualTo(Response.Status.OK.getStatusCode());
+
+        List<String> providerEntityIds = response.readEntity(new GenericType<List<String>>() {
+        });
+        assertThat(providerEntityIds).containsOnly(
+                ENABLED_ALL_RP_IDP,
+                ENABLED_FOR_IDP_RESPONSE_SOFT_DISCONNECTING_IDP
+        );
+    }
+
     @Test
     public void getEnabledIdentityProviderEntityIdsForSignIn_returnsOkAndIdps() {
         String entityId = "not-test-rp";
         Response response = getIdpListForLoA(entityId, LevelOfAssurance.LEVEL_1, Urls.ConfigUrls.ENABLED_ID_PROVIDERS_FOR_SIGN_IN_RESOURCE);
         assertThat(response.getStatus()).isEqualTo(Response.Status.OK.getStatusCode());
 
-        List<String> providerEntityIds = response.readEntity(new GenericType<List<String>>(){});
+        List<String> providerEntityIds = response.readEntity(new GenericType<List<String>>() {
```

Comment:
If these formatting changes really needed to happen, I'd have preferred them in a separate commit, but no big deal. It looks neater on one line, FWIW.

Reply:
Fair, I was being lazy....